backend/kernelCI_app/views/treeView.py

```diff
@@ -40,7 +40,11 @@ def get(self, request):
             )
 
             SELECT
-                checkouts.*,
+                checkouts._timestamp, checkouts.id, checkouts.origin, checkouts.tree_name,
+                checkouts.git_repository_url, checkouts.git_repository_branch, checkouts.patchset_files,
+                checkouts.patchset_hash, checkouts.message_id, checkouts.comment, checkouts.start_time,
+                checkouts.contacts, checkouts.log_url, checkouts.log_excerpt, checkouts.valid, checkouts.misc,
+                CONCAT(checkouts.git_commit_hash, '/', checkouts.git_commit_name) AS git_commit_hash,
```

Comment:
I think that we want to show the tag **instead** of the commit hash when we have a valid one.

Reply:
yes, that is the idea for now